0001-data-collection-extension/README.md

```diff
@@ -74,7 +74,7 @@ The recommended solution is to save server timestamp upon network message receiv
 ### Request/response
 The extension uses request/response pattern. A response is sent if a bucket requires a processing confirmation.
 
-For MQTT, responses MUST be published at `<request_path>/status`.
+For MQTT, responses MUST be published at `<request_path>/status`. Each response MUST be published with the same `QoS` as the corresponding request.
```

Comment:
QoS is not a piece of code or something like that, so it should be plain text, not `monospace`.